Context.sublime-menu

```diff
@@ -2,9 +2,12 @@
     {
         "caption": "Sublack: Format file",
         "command": "black_file"
-    },    
+    },
     {
         "caption": "Sublack: Diff file",
         "command": "black_diff"
     },
+    {
+        "command": "black_toggle_black_on_save"
+    },
```

Comment:
3 is many for the menu.  we should turn all of this in a sublack submenu

Reply:
I think 3 is okay. It's also very accessible this way. (I don't have lots of stuff in the context menu actually.)

I wouldn't know how to do this, and I would only consider this if users complain via opening issues.

Reply:
I must say I never use this submenu... we'll see